Sources/Paywalls/PaywallCacheWarming.swift

```diff
@@ -90,6 +90,15 @@ actor PaywallCacheWarming: PaywallCacheWarmingType {
 
         Logger.verbose(Strings.paywalls.warming_up_images(imageURLs: imageURLs))
 
+        // Preferred method - load with FileRepository
+        let fileRepository = FileRepository()
+        for url in imageURLs {
+            Task {
+                _ = try await fileRepository.generateOrGetCachedFileURL(for: url)
+            }
```

Comment:
With images, we may want to consider using a TaskGroup because there could be so many image URLs to process. Though we would need to make sure that we configure the task group not to bail if one fails.

This can be an improvement in a future PR though.